packages/dicomImageLoader/src/imageLoader/wadors/metaData/metaDataProvider.ts

```diff
@@ -266,6 +267,28 @@ function metaDataProvider(type, imageId) {
       actualFrameDuration: getNumberValue(metaData['00181242']),
     };
   }
+
+  // This is used for gathering all the metadata for export
+  if (type === 'dataset') {
+    const types = [
```

Comment:
Can you externalize this list of types, and add a comment on it?  It definitely is NOT all the types, but there doesn't seem to be a safe way to get all the actual types available.  Would be nice to have a way for a module to register a new type if needed.